tt_eager/tt_dnn/op_library/eltwise_unary/eltwise_unary_op.hpp

```diff
@@ -281,7 +281,7 @@ constexpr auto div_unary_sfpu = make_eltwise_asymmetric_binop_unary_with_param<U
 
 inline Tensor exp(
     const Tensor& input_tensor,
-    bool fast_and_approx = false,
+    bool fast_and_approx,
```

Comment:
Why was the default value removed?

Reply:
Because there are 2 exp functions. This one and another one that doesn't take this `bool`. And because both functions had defaults on every argument except `input_tensor`, they were ambiguous when only `input_tensor` was being passed in. So, I removed the default value from this overload. And now if `exp` is only called with `input_tensor` it will use the other overload which uses this version with `fast_and_approx=false`